src/main/java/ca/on/oicr/pineryreports/util/SampleUtils.java

```diff
@@ -144,6 +146,19 @@ public static boolean isCreatedBetween(SampleDto dto, String start, String end)
     return dto -> getAttribute(ATTR_RECEIVE_DATE, dto) == null;
   }
 
+  public static Predicate<SampleDto> byDistributed() {
+    return byDistributedBetween(null, null);
+  }
+
+  public static Predicate<SampleDto> byDistributedBetween(String startDate, String endDate) {
+    return dto -> {
+      String distributionDate = getAttribute(ATTR_DISTRIBUTION_DATE, dto);
+      return distributionDate != null
+          && (startDate == null || distributionDate.compareTo(startDate) > 0)
```

Comment:
If this is meant to use the start/end dates provided for the report, the `startDate` condition should be inclusive (`>= 0`)

Reply:
And so it is! Thanks